active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,591 @@
+# Teaching tip
+
+## Description
+
+A teaching tip is a content-rich nonmodal flyout that provides contextual information. A teaching tip looks like a flyout, except that it may target a UI element with a pointer or be persistent and require the user to manually dismiss it. A teaching tip is often used for informing, reminding, and teaching users about important and new features that may enhance their experience. 
+
+**Important APIs:** [TeachingTip class]() [Link TODO]
+
+A teaching tip may also be light-dismiss enabled or used without a pointer indicating reference to any specific UI element. 
+
+
+### Is this the right control? 
+
+Use a **TeachingTip** control to inform users about new and important updates and features, focus users on a component to advise them about its options or state, remind users of nonessential options that would improve their experience, or teach users how a task should be completed. 
```

Comment:
Suggestion: "focus users' attention"

Reply:
```suggestion
Use a **TeachingTip** control to focus a user's attention on new or important updates and features, remind a user of nonessential options that would improve their experience, or teach a user how a task should be completed. 
```

Reply:
@YuliKl Does this feel more clear and succinct?

Reply:
Yes, sounds great to me.